src/controllers/playback/videoosd.js

```diff
@@ -1388,8 +1392,16 @@ define(["playbackManager", "dom", "inputManager", "datetime", "itemHelper", "med
 
                 case "mouse":
                     if (!e.button) {
-                        playbackManager.playPause(currentPlayer);
-                        showOsd();
+                        if (playerPauseClickTimeout) {
+                            clearTimeout(playerPauseClickTimeout);
+                            playerPauseClickTimeout = 0;
+                        } else {
+                            playerPauseClickTimeout = setTimeout(() => {
+                                playbackManager.playPause(currentPlayer);
+                                showOsd();
+                                playerPauseClickTimeout = 0;
```

Comment:
This shouldn't be required either. The ids should not be reused, so we shouldn't need to reset it when complete. https://developer.mozilla.org/en-US/docs/Web/API/WindowOrWorkerGlobalScope/setTimeout#Return_value

Reply:
oh wow, I've been living a lie lol. Thanks for letting me know thornbill!